differential/customfield/__tests__/DifferentialBugzillaIdCustomFieldTestCase.php

```diff
@@ -6,28 +6,22 @@
 final class DifferentialBugzillaBugIDCustomFieldTestCase
   extends PhabricatorTestCase {
 
-  public function testBugIdCommitMessageFieldValidation() {
-    $test = array(
-      'false' => array(null, '', '12.3', '1e3'),
-      'true' => array('123')
-    );
+  public function testBugIdCommitMessageFieldBasicValidation() {
+    $invalids = array(null, '', ' ', '12.3', '1e3');
```

Comment:
Are the values "T4567", "aaaa", or ";" valid?

Reply:
Negative.

Reply:
Did you test it out?